packages/mdc-menu/foundation.ts

```diff
@@ -82,7 +82,7 @@ export class MDCMenuFoundation extends MDCFoundation<MDCMenuAdapter> {
     const isTab = key === 'Tab' || keyCode === 9;
 
     if (isTab) {
-      this.adapter_.closeSurface();
+      this.adapter_.closeSurface(true);
```

Comment:
nit: `this.adapter_.closeSurface(/** restoreFocus */ true);`

This'll document the boolean value.